src/expr.rs

```diff
@@ -674,9 +674,13 @@ impl<'a> Rewrite for Loop<'a> {
             ControlBraceStyle::AlwaysNextLine => alt_block_sep.as_str(),
             ControlBraceStyle::AlwaysSameLine => " ",
         };
+        let block_width = try_opt!(width.checked_sub(label_string.len() + self.keyword.len() +
+                                                     extra_offset(&pat_expr_string, inner_offset) +
+                                                     1));
```

Comment:
This seems right for the empty loop case, but wrong when we have statements in the loop. In that case, the width for the statement should be just the width minus the next block offset (which block.rewrite will take care of). Or am I misunderstanding what is going on here?